NIPTool/API/external/api/deps.py

```diff
@@ -67,20 +62,18 @@ def get_password_hash(password):
     return pwd_context.hash(password)
 
 
-def authenticate_user(username: str, password: str) -> Optional[UserInDB]:
+def authenticate_user(username: str, password: str) -> Optional[User]:
```

Comment:
Perhaps better to raise if not found? Also consider communicating if user is not found or pwd is wrong

Reply:
Insteead of raising exeption I want to be redirected to the index page.  I have a problem with this though. Hope we can talk about it tomorrow